rfcs/010-react-intl.md

```diff
@@ -0,0 +1,53 @@
+# RFC 010 - Best Practices with react-intl
+
+## Affected projects
+
+- Frontend
+
+## Motivation
+
+We're using react-intl and have currently 2 ways of expressing a message ("defineMessages + formatMessage", or "FormattedMessage"). It's currently not clear when to use one or another approach and guidelines would be useful for current and new developers.
+
+## Comparaison of the 2 approaches
+
+### defineMessages + formatMessage
+
+Pros:
+- Without JSX "bloat", concise
+- Can be assigned to a variable
+- Can be post manipulated in Javascript
+- Support values
```

Comment:
```suggestion
- Support values
- Because it exports a string, it's sometimes the only option available (ie. to fill an HTML `select`'s options)
```